virtcontainers/qemu.go

```diff
@@ -152,6 +153,11 @@ func (q *qemu) kernelParameters() string {
 	// set the maximum number of vCPUs
 	params = append(params, Param{"nr_cpus", fmt.Sprintf("%d", q.config.DefaultMaxVCPUs)})
 
+	// Add a kernel param to indicate if vsock is being used.
+	// This will be consumed by the agent to determine if it needs to listen on
+	// a serial or vsock channel
+	params = append(params, Param{vsockKernelOption, strconv.FormatBool(q.config.UseVSock)})
```

Comment:
Just one comment as I want to make sure we don't override the behavior of the runtime here. Is `q.config.UseVSock` the value coming directly from the `configuration.toml` file? Or is it coming from the combination of the config file and the host capability?

Reply:
@sboeuf It is a combination of the config file and the host capability.

Reply:
Perfect!